### ISSUE TITLE
Clarify language in installation prompts

### ISSUE THREAD

**Issue by lightnin** [YunoHost-Apps/pleroma_ynh#196]
As described here:
https://github.com/YunoHost-Apps/pleroma_ynh/issues/133

Pleroma installer asks:
 "en": "Is it a public application?",
Since it is necessary for pleroma to be a public application for federation to work, which is a core feature of the app, I suggest we remove this feature and automatically make all new installations "A public application" without asking the user to indicate this.
Installer also asks:

 "en": "Is the registration open for new users?"
This should be clarified to resolve questions described in 133, for example:
"Should registration be open to users who do not have a Yunohost account on the system?"

**Comment by yalh76** [comment]
> Pleroma installer asks:
> "en": "Is it a public application?",
> Since it is necessary for pleroma to be a public application for federation to work, which is a core feature of the app, I suggest we remove this feature and automatically make all new installations "A public application" without asking the user to indicate this.

I disagree, someone could install a pleroma instance without federation ...

> Installer also asks:
"en": "Is the registration open for new users?"
This should be clarified to resolve questions described in 133, for example:
"Should registration be open to users who do not have a Yunohost account on the system?"
It might be worth clarifying that allowing registration from non-yunohost users will make it possible for spammers to create accounts on the system.


Done at https://github.com/YunoHost-Apps/pleroma_ynh/commit/747c2b9ecb0882fb33a3187926107be1a47da4a6

**Comment by artlog** [comment]
@lightnin 
As @yalh76 stated , first request won't be fixed : this can still be installed without federation. Even for testing or demonstration purposes it is relevant to leave this configurable.
And second was fixed.

So closing this Issue.

**Comment by artlog** [comment]
If closure of this Issue should be discussed forum might be better ( https://forum.yunohost.org/c/discuss/ )
